src/skaero/util/decorators.py

```diff
@@ -5,9 +5,9 @@
 
 """
 
-from __future__ import division, absolute_import
+from __future__ import absolute_import, division
```

Comment:
Don't remember what's the Python 2/3 status of scikit-aero, but if you are not supporting Python 2, there's no need of `__future__` imports at all.